src/v/cluster/topic_table.h

```diff
@@ -79,6 +79,23 @@ class topic_table {
 public:
     enum class topic_state { exists, not_exists, indeterminate };
 
+    // Guide to various partition revisions (i.e. offsets of the corresponding
```

Comment:
:+1:

There is also this initial_revision I don't fully understand.

Reply:
It's related to recovery from cloud storage. Something like the revision of the topic that we recovered from or the creation revision of the topic if none? Need to look it up as well :)